ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js

```diff
@@ -75,17 +77,10 @@ const ConfirmAddCustodianToken = () => {
             await dispatch(setProviderType(networkType));
           }
 
-          let custodianName = connectRequest.service.toLowerCase();
-
-          if (connectRequest.service === 'JSONRPC') {
-            custodianName = connectRequest.environment;
```

Comment:
I'm pretty certain from looking at the config, that in every single case environment is correct, as it seems to always be the name with lowercase.

But we should revert this change if that's not the case.